static/app/views/releases/detail/overview/issues.tsx

```diff
@@ -78,13 +126,18 @@ class Issues extends Component<Props, State> {
         query.setTagValues('error.handled', ['0']);
         break;
       case IssuesType.RESOLVED:
+        query.setTagValues('release', [version]);
+        query.setTagValues('is', ['resolved']);
+        break;
```

Comment:
I'm afraid this is not possible in Issues. We would want to see issues resolved by this release. Not issues happening in this release that are currently resolved.